SUMMARY.md

```diff
@@ -14,4 +14,5 @@
 
 * [Slides](lectures/lecture-01/slides.md)
 * [Feedback](lectures/lecture-01/feedback.md)
-* [Codelab 1](lectures/lecture-01/codelab.md)
+* [Codelab 1](codelabs/codelab-01/README.md)
```

Comment:
Break this up by date, see the Summary here: https://github.com/UMD-CS-STICs/389Lfall17/blob/master/SUMMARY.md

(You can see the raw version on GitHub, too, just click "Raw")